html_tag_collector/collector.py

```diff
@@ -1,3 +1,4 @@
+from dataclasses import asdict
```

Comment:
Add a docstring describing the function of `collector.py` here at the top of the function. This will make it easier for people in the 🤖 _future_ 🤖 to understand at a glance what this module is doing.

Reply:
Good description?

```python
""" The tag collector is used to collect HTML tags and other relevant data from websites that is useful for training prediction models.
    Information being collected includes:
        - The URL's path
        - HTML title
        - Meta description
        - The root page's HTML title
        - HTTP response code
        - Contents of H1-H6 header tags
        - Contents of div tags
"""
```